actions/views/rhs.ts

```diff
@@ -441,8 +446,13 @@ export function closeRightHandSide() {
                 channelId: '',
                 timestamp: 0,
             },
-            unsetEditingPost(),
-        ]));
+        ];
+
+        if (editingPost?.isRHS) {
+            actionsBatch.push(unsetEditingPost());
```

Comment:
Regarding

> > Is there a test we can write for this?
> 
> mmh, good question. I think other than testing what actions fire (I am currently adjusting that) and the state changes is probably not possible.

That's actually a benefit of doing it this way. You can test that the reducer causes that one change to happen when the RHS is closed without having to care about how it might affect the rest of the state